### ISSUE TITLE
network diverges

### ISSUE THREAD

**Issue by loyeamen** [fchollet/keras#932]
Hi,

It seems that using 'rmsprop' and 'categorical_crossentropy' the network diverges. 
370784/370784 [==============================] - 6700s - loss: 1.8887 - acc: 0.4243 - val_loss: 1.8804 - val_acc: 0.4097
370784/370784 [==============================] - 6702s - loss: 1.9328 - acc: 0.4118 - val_loss: 1.9089 - val_acc: 0.3977
370784/370784 [==============================] - 6702s - loss: 1.9986 - acc: 0.3866 - val_loss: 1.9611 - val_acc: 0.3873
370784/370784 [==============================] - 6708s - loss: 2.0482 - acc: 0.3752 - val_loss: 1.9454 - val_acc: 0.3878
370784/370784 [==============================] - 6714s - loss: 2.1057 - acc: 0.3539 - val_loss: 2.0767 - val_acc: 0.3591
137856/370784 [==========>...................] - ETA: 4147s - loss: 2.1578 - acc: 0.3390

I use a convolutional network with dropout on the last FC layers. Can dropout cause to such a behaviour, or it just a bug?

Thanks,

Oren
